packages/select-menu/src/types.ts

```diff
@@ -0,0 +1,43 @@
+export type SelectMenuAppearanceType = 'primary' | 'minimal' | 'card'
+
+export type SelectMenuChildrenFn<T extends OptionBase> = <T>(props: {
+  toggle: () => void
+  getRef: (ref: any) => void
+  isShown: boolean
+  selectedItems: T | T[]
+}) => React.ReactNode
+
+export interface SelectMenuProps<T extends OptionBase> {
+  options: T[]
+  value: string | string[]
+  children: React.ReactNode | SelectMenuChildrenFn<T>
+  appearance?: SelectMenuAppearanceType
+  hasFilter?: boolean
+  hasTitle?: boolean
+  title?: string
+  width?: number
+  height?: number
```

Comment:
actually, height doesn't change anything in popover behavior - instead, it should limit popover's height (using max-height) to certain constraint and make it scrollable